_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Introducing Security Analytics"
+authors:
+  - christophermoore
+date:   2023-03-13 12:15:00 -0700
```

Comment:
Please contact me to reschedule the release date for this blog.

Reply:
@jimishs - when do you plan to push this live? I have tentatively scheduled it for 5/1/23.

Reply:
@jimishs - Please advise on the schedule for completion for this blog. It's overdue on the publishing date.